enterprise_access/apps/subsidy_access_policy/customer_api.py

```diff
@@ -0,0 +1,48 @@
+"""
+Python API for interacting with the lms client
+for use in the domain of SubsidyAccessPolicies.
+"""
+import logging
+
+from django.conf import settings
+from edx_django_utils.cache import TieredCache
+from requests.exceptions import HTTPError
+
+from enterprise_access.apps.api_client.lms_client import LmsApiClient
+from enterprise_access.cache_utils import versioned_cache_key
+
+logger = logging.getLogger(__name__)
+
+DEFAULT_CACHE_TIMEOUT = getattr(settings, 'LMS_CLIENT_CACHE_TIMEOUT', 60 * 5)
```

Comment:
minor, non-blocking suggestions:
1. Set a default value for this setting in `settings/base.py`. 5 minutes is fine.  There's already a [section for cache timeouts](https://github.com/openedx/enterprise-access/blob/75363c6683e7e09f5dfb9ffd3ba5909584a2416a/enterprise_access/settings/base.py#L497).
2. Consider making the setting name more specific, e.g. `ENTERPRISE_USER_RECORD_CACHE_TIMEOUT`.